jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/datadisplay/IconsTests.java

```diff
@@ -0,0 +1,34 @@
+package io.github.epam.material.tests.datadisplay;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.*;
+
+public class IconsTests  extends TestsInit {
+
+    @Test
+    public void simpleIconsTest() {
+        simpleIconsPage.open();
+
+        iconsFrame.filled1.is().displayed();
+        iconsFrame.filled2.is().displayed();
+        iconsFrame.oulined1.is().displayed();
+    }
+
+    @Test
+    public void colorIconsTest() {
```

Comment:
Can we check the color of the icon in this case?

Reply:
I added check for color